source/Classroom/Basics/API/does_sendgrid_offer_a_sandbox_or_test_environment.md

```diff
@@ -11,16 +11,18 @@ navigation:
   show: true
 ---
 
-We do not offer a true "sandbox" environment, however we do have a [free trial](https://sendgrid.com/transactional-email/pricing) which allows you to send 100 emails per day, and also gives you access to almost all of the features offered with our higher package levels. The free trial does not include access to a dedicated IP or IP whitelabeling.
+SendGrid offers a [free trial](https://sendgrid.com/transactional-email/pricing) which allows you to send 100 emails per day, and also gives you access to almost all of the features offered with our higher package levels. The free trial does not include access to a dedicated IP or IP whitelabeling.
 
-We also have a [sink email address]({{root_url}}/Classroom/Troubleshooting/Delivery_Issues/safely_test_your_sending_speed.html) that can help you test message throughput for your integration, as well as a [sandbox_mode setting]({{root_url}}/Classroom/Send/v3_Mail_Send/sandbox_mode.html) for API v3 which prevents emails from being sent to the recipient. 
+We have a [sink email address]({{root_url}}/Classroom/Troubleshooting/Delivery_Issues/safely_test_your_sending_speed.html) that can help you test message throughput for your SMTP integration. 
 
-{% info %}
-SendGrid is in the process of introducing a 30 day Free Trial alongside an improved onboarding experience.
+If you are looking to test you API sending integration, you can enable the sandbox_mode in the [mail_settings parameters of our Mail Send API](https://sendgrid.com/docs/Classroom/Send/v3_Mail_Send/sandbox_mode.html). **This will allow you to validate each parameter you include, and the structure of your JSON payload, all without actually sending a message.**
```

Comment:
**Sandbox Mode allows you to validate each included parameter and the structure of your JSON payload - all without sending a message.**